Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1833,78 +1835,155 @@ inline void SetImage(CGImageRef image) {
 
 /**
  @Status Caveat
- @Notes Limited bitmap formats available. Decode, shouldInterpolate, intent parameters
- and some byte orders ignored.
- */
+ @Notes If the image colorspace is not a valid render target format (as per
+ https://msdn.microsoft.com/en-us/library/windows/desktop/dd756766(v=vs.85).aspx#supported_wic_formats), it's converted to 32bpp BGRA
+ format. Does not support conversion if data is already provided.
+*/
```

Comment:
Could we instead document the CG bitmap formats that we can indeed support, we have already done the mapping and calling it out here will make it easier for anyone using the code.  Also, if we don't support the specified format, please return null instead of silently picking a different format.